packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -267,10 +290,9 @@ describe(testCase, () => {
         endorsingOrgs: ["org1.example.com", "Org2MSP"],
       };
 
-      const res = await apiClient.runTransactionV1(req);
+      const res = await pluginWithSerializedInputs.transact(req);
```

Comment:
@jagpreetsinghsasan Please also leave in the test for the API client not just through direct invocation. Otherwise we might lose coverage. It's important to verify that the same operation works through the API too not just via direct invocation so that we can continue to be "API first"